lib/modules/syntaxHighlighting.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+
+import hljs from 'highlight.js';
+import { Module } from '../core/module';
+import { watchForElements, watchForThings } from '../utils';
+
+export const module: Module<*> = new Module('syntaxHighlighting');
+
+module.moduleName = 'syntaxHighlightingName';
+module.category = 'appearanceCategory';
+module.description = 'syntaxHighlightingDesc';
+module.disabledByDefault = false;
+
+module.beforeLoad = () => {
+	watchForElements(['selfText'], null, ele => hljs.highlightElement(ele.querySelector('pre code')));
+	watchForThings(['comment', 'message'], thing => hljs.highlightElement(thing.entry.querySelector('pre code')));
```

Comment:
After taking a closer look, it is probably possible to cover all cases by simply using:
`watchForElements(['selfText', 'page'],  'pre code', hljs.highlightElement)`